### DISCUSSION TITLE
Data Subscription Limits

### DISCUSSION THREAD

**Discussion by tehkyle** [ravendb/ravendb#13587] [General]
Is there a maximum number of data subscriptions that can be created on a cluster? If not, is there a recommended 'soft-max'?

**Comment by garayx** [comment]
Hi

Yes, the maximum amount is 1000 connections.
We have this documented here: [https://ravendb.net/docs/article-page/5.3/csharp/server/configuration/subscription-configuration#subscriptions.maxnumberofconcurrentconnections](https://ravendb.net/docs/article-page/5.3/csharp/server/configuration/subscription-configuration#subscriptions.maxnumberofconcurrentconnections)

**Reply by ayende** [reply]
Please note, the limit is to the number of concurrent subscriptions that are _in use_. Also, note that this is the limit of the concurrent subscription on a _single node_. So in a cluster, you can have more globally.

What scenario do you have where you may have that many active at one point?
